legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py

```diff
@@ -866,6 +867,30 @@ def test_unpaid_filing(session, client, jwt):
                     }
       },
      HTTPStatus.OK, None
+     ),
```

Comment:
can we add similar tests for the other corp legal types?  i.e. BC, CC, ULC

Reply:
I didn't add similar tests for the other types because the code will check for these four:
`{'types': 'BC,BEN,ULC,CC', 'outputs': ['noticeOfArticles', 'certificate']},`
I was thinking along the same lines as if you had a test for even numbers, I wouldn't put tests for 2, 4, 6, 8, etc.
Do you think it's necessary to test all four?

Reply:
yes, I think we should. as soon as you put in specific logic for specific legal types, it may detect failing scenarios

Reply:
I added tests for all four company types.

Reply:
thanks, looks good 👍